lib/krane/kubernetes_resource/daemon_set.rb

```diff
@@ -59,7 +59,9 @@ def relevant_pods_ready?
       relevant_node_names = @nodes.map(&:name)
       considered_pods = @pods.select { |p| relevant_node_names.include?(p.node_name) }
       @logger.debug("Considered #{considered_pods.size} pods out of #{@pods.size} for #{@nodes.size} nodes")
```

Comment:
I know you didn't touch this line, but maybe we should add `rollout_data["numberReady"]` to this debug line so that if this causes an unexpected regression for daemonset tolerance, we can debug it more easily